ui/src/app/base/components/ActionButton/ActionButton.md

```diff
@@ -0,0 +1,21 @@
+## Default
+
+```jsx
+<ActionButton appearance="positive">Click me!</ActionButton>
+```
+
+## Loading
+
+```jsx
+<ActionButton appearance="positive" loading={true}>
```

Comment:
all the props set to `{true}` can just be the props themselves, e.g.

`<ActionButton appearance="positive" loading>`
`<Button appearance="neutral" disabled>`

etc.

Reply:
Done.